domain/src/main/kotlin/app/discord/user/entity/User.kt

```diff
@@ -1,13 +1,23 @@
 package app.discord.user.entity
 
+import app.discord.attendance.dto.UserAttendanceHistory
+import app.discord.attendance.entity.Attendance
+import java.time.OffsetDateTime
+
 class User (
-    val id: String,
+    val guildId: String,
+    val userId: String,
     val username: String,
     val globalName: String,
-    val authId: Int,
-    val enabled: Boolean,
-    val firstJoinTimestamp: Long,
-    val guildOutTimestamp: Long,
-    val lastActivityTimestamp: Long
+    val nickName: String,
+    val registerTime: OffsetDateTime,
+    val leaveTime: OffsetDateTime,
+    val isBan: Boolean,
+    userAttendanceHistory: UserAttendanceHistory
 ){
-}
+    private val attendance: Attendance
+    init{
+        attendance = Attendance(histories = userAttendanceHistory)
+    }
```

Comment:
별다른 이유가 없다면 init 블록 없이 합쳐서 쓰는 쪽이 중복코드를 줄일 수 있을것 같습니다.
```suggestion
    private val attendance: Attendance = Attendance(histories = userAttendanceHistory)
```